_plugins/jekyll_copy_uswds.rb

```diff
@@ -0,0 +1,12 @@
+module Jekyll
+  class JekyllCopyUswds < Generator
+    def generate(site)
+      source = "node_modules/uswds/dist"
+      Dir.glob(File.join(source, "{fonts,img}", "**", "*")) do |filename|
+        next if File.directory?(filename)
+        dir, name = File.split(filename.delete_prefix(source))
+        site.static_files.push(Jekyll::StaticFile.new(site, source, dir, name))
+        end
+    end
+  end
+end
```

Comment:
Here's the magic (aka the results of lots of Googling). This "generator" runs as part of the Jekyll build. It includes all USWDS fonts and images as static files to be included in the build. They are output directly in the output folder. Since this is part of the Jekyll build, we no longer need an extra step to manage this.

If we decide to remove the Node dependency completely, this would need to be updated accordingly.